demos/noise_suppression_demo/cpp/main.cpp

```diff
@@ -2,67 +2,49 @@
-
-#include "openvino/openvino.hpp"
-
-#include "gflags/gflags.h"
-#include "utils/common.hpp"
-#include "utils/slog.hpp"
-
-typedef std::chrono::steady_clock Time;
-
-static const char help_message[] = "Print a usage message.";
-static const char inp_wav_message[] = "Required. Path to a input WAV file.";
-static const char out_wav_message[] = "Optional. Path to a output WAV file.";
-static const char model_message[] = "Required. Path to an .xml file with a trained model.";
-static const char target_device_message[] = "Optional. Specify the target device to infer on (the list of available "
-                                            "devices is shown below). Default value is CPU. "
-                                            "The demo will look for a suitable plugin for device specified.";
-DEFINE_bool(h, false, help_message);
-DEFINE_string(i, "", inp_wav_message);
-DEFINE_string(o, "noise_suppression_demo_out.wav", out_wav_message);
-DEFINE_string(m, "", model_message);
-DEFINE_string(d, "CPU", target_device_message);
-
-static void showUsage() {
-    std::cout << std::endl;
-    std::cout << "noise_suppression_demo [OPTION]" << std::endl;
-    std::cout << "Options:" << std::endl;
-    std::cout << std::endl;
-    std::cout << "    -h           " << help_message << std::endl;
-    std::cout << "    -i INPUT     " << inp_wav_message << std::endl;
-    std::cout << "    -o OUTPUT    " << out_wav_message << std::endl;
-    std::cout << "    -m MODEL     " << model_message << std::endl;
-    std::cout << "    -d DEVICE    " << target_device_message << std::endl;
-}
-
-bool ParseAndCheckCommandLine(int argc, char* argv[]) {
-    gflags::ParseCommandLineNonHelpFlags(&argc, &argv, true);
-    if (FLAGS_h) {
-        showUsage();
-        return false;
+#include <utils/common.hpp>
+#include <utils/slog.hpp>
+
+#include <gflags/gflags.h>
+#include <openvino/openvino.hpp>
+
+using namespace std;
+
+namespace {
+constexpr char h_msg[] = "show the [H]elp message and exit";
+DEFINE_bool(h, false, h_msg);
+
+constexpr char m_msg[] = "path to an .xml file with a trained [M]odel";
+DEFINE_string(m, "", m_msg);
+
+constexpr char i_msg[] = "path to an [I]nput 16kHz WAV file";
+DEFINE_string(i, "", i_msg);
+
+constexpr char d_msg[] = "specify a [D]evice to infer on (the list of available devices is shown below)";
+DEFINE_string(d, "CPU", d_msg);
+
+constexpr char o_msg[] = "path to an [O]utput WAV file";
+DEFINE_string(o, "noise_suppression_demo_out.wav", o_msg);
+
+void parse(int argc, char *argv[]) {
+    gflags::ParseCommandLineFlags(&argc, &argv, false);
+    slog::info << ov::get_openvino_version() << slog::endl;
+    if (FLAGS_h || 1 == argc) {
+        cout <<   "\t[-h]                                  " << h_msg
+             << "\n\t[--help]                              print [HELP] on all arguments"
+             << "\n\t -m <[M]odel file>                    " << m_msg
+             << "\n\t[-i] <[I]nput file>                   " << i_msg
```

Comment:
I stole the idea from key bindings. To refer to a bound key they make the corresponding letter capital and highlight with color. For our args it should make it easier to remember.

For python this gives a hint that one letter args are also possible.

Reply:
but in C++ case there will be no highlighting with bold or color, so will look a bit strange?

Reply:
Thus I use []

Reply:
right, and what is the meaning of `[]`, is that optional parameters? Not obvious.

Reply:
For `[-d]` (I just changed -i to be required) [] stand for optional. For regular words [] is highlighting

Reply:
highlighting on github only? @andrew-zaytsev are you ok if we add github specific highlighting to demo option description? Will this also work for openvino online documentation?

Reply:
no. These are regular chars)

Reply:
I did not get your statement: "For regular words [] is highlighting". You make this in C++ code, not in markdown, where it will be highlighted?

Reply:
Highlighting here means "ease of noticing"

Reply:
I still think that it makes more harm than brings profit, at least for C++. Those brackets make text less readable. For python it might be OK, as they highlight short option name, for C++ it is useless as there's only one name for each option. If it ain't broke - don't fix it.

Reply:
Removed